projects/netgrif-components-core/src/lib/actions/model/front-action-definition.ts

```diff
@@ -0,0 +1,6 @@
+import {Injector} from "@angular/core";
+import {FrontAction} from "../../data-fields/models/changed-fields";
+
+export interface FrontActionDefinition {
+    fn: (injector: Injector, frontAction: FrontAction) => void;
```

Comment:
I would prefer a different name for the action function, if possible `call`.

Reply:
Done.